### DISCUSSION TITLE
Not able to install Windows 11 x64 on M1 arm using v3.0.4

### DISCUSSION THREAD

**Discussion by PhotoNomad0** [utmapp/UTM#3580] [UTM Help (macOS)] [answered]
On 3.0.4 I am trying to install Windows 11 x64 (intel version) on an M1 MBP.  This is Wanted for SW testing. I have followed these steps to disable checking: https://www.repairwin.com/how-to-install-windows-11-on-virtualbox-even-without-tpm/ .  Initially it seems to be working, but after reboot it starts giving various errors and installation never completes.

Has anyone gotten this to work?

**Comment by PhotoNomad0** [comment, marked as answer]
Had success getting Windows 11 x64 (intel version) to run on an M1 MBP.  But is was a bit involved as I created the VM on an Intel mac:
- started by installing UTM on an Intel based Mac.  Created a new virtualized machine, selected windows as OS and used the Windows 11 installation ISO.  Used the steps in https://www.repairwin.com/how-to-install-windows-11-on-virtualbox-even-without-tpm/
- then went into the settings for the VM and turned off `Use hypervisor` on the QEMU settings.
- Then I copied the VM to the M1 MBP.
- then on the M1 MBP, I went into the settings for the VM:
  - under system I selected `show advanced settings`.
  - then set CPU Cores to 4 and turned on `Force Multicore`
  - clicked save.

It then started up - though very slowly the first time, and on following boots it's a bit faster (emulation is not fast).

later I tried changing the display settings from `virtio-vga` display to  `spice QXL GPU (primary, vga compatible) (qxl vga)`, which seems faster?  

Is there a better display setting for emulated Intel x64 running on M1 arm?

Anyway it does seem functional as a test VM.

**Reply by Guneetgstar** [reply]
> * Then I copied the VM to the M1 MBP.
> * then on the M1 MBP, I went into the settings for the VM:
>   
>   * under system I selected `show advanced settings`.
>   * then set CPU Cores to 4 and turned on `Force Multicore`
>   * clicked save.

Can you please share VM on a drive if you have a copy? I have been trying to install x86 emulation on my M1 for a month but no success yet.

**Comment by jgstew** [comment]
It doesn't work for everything, but I found that running Windows 11 ARM edition on M1 and then using Windows 11 ARM's built in Intel/AMD x64 emulation for running x64 programs to be much faster.

**Reply by PhotoNomad0** [reply]
I agree, but for `testing` Windows x64 programs and for building older programs, running the x64 version is better since there are a few compatibility problems with the Arm64 version. But the tradeoff is that it is very slow.

**Comment by mmesnjak** [comment]
Could someone please share all tweaks needed to be done on VM configuration before installing x64 Windows 11 using emulation in UTM on Mac M1 Max with Sonoma 14.1.1? 

I ask because I cannot finish my installation successfully.
I am using Win11_23H2_English_x64.iso downloaded from Microsoft site, and it just gets to the point with blue screen saying that it needs to reboot.

Thank You!
